scripts/irods/test/test_resource_types.py

```diff
@@ -4433,6 +4433,29 @@ def count_good_and_bad(ils_l_output):
         self.assertEqual(num_out_of_date, 0)
         os.unlink(filename)
 
+    def test_replicate_to_consumer_repl_hier_from_provider__4319(self):
+        filename = 'test_replicate_to_consumer_repl_hier_from_provider__4319'
+        lib.make_file(filename, 3 * 1024 * 1024)
```

Comment:
1 * 1024 * 1024 + 1 (as minimal test?)

Reply:
Will do. CI did not fail as expected without the fix commit, so I am looking into this test.